app/services/feature_flag.rb

```diff
@@ -1,9 +1,31 @@
+# This module provides mechanisms for toggling on and off features.
+#
+# @note A wrapper around the Flipper gem
 module FeatureFlag
   class << self
     delegate :add, :disable, :enable, :enabled?, :exist?, :remove, to: Flipper
 
+    # Unless the given :feature_flag_name is _explicitly_ disabled,
+    # this method returns true.
+    #
+    # @param feature_flag_name [Symbol]
+    # @param args [Array] passed to FeatureFlag.enabled?
+    #
+    # @return [TrueClass] go ahead and use this feature
+    # @return [FalseClass] don't use this feature
+    #
+    # @note This is an optimistic test, namely if the given
+    #       :feature_flag_name does not exist (e.g., has never been
+    #       enabled, enabled, or has been removed), the feature is
```

Comment:
**Quick question/suggestion:** did you mean to write "disabled" instead of "enabled" here? If not, we can remove the second "enabled" altogether.
```suggestion
    #       enabled, disabled, or has been removed), the feature is
```